Fix version of GPL mentioned in about box to match what's in the README
and the source code headers.

https://sourceforge.net/p/gelide/bugs/22/
--- src/ui/dialog_about.cpp.orig	2011-03-25 13:46:23.000000000 -0500
+++ src/ui/dialog_about.cpp	2024-07-23 04:39:47.000000000 -0500
@@ -52,7 +52,7 @@
 		"\n"
 		"You may redistribute it and/or modify it under the terms of the\n"
 		"GNU General Public License, as published by the Free Software\n"
-		"Foundation; either version 2 of the License, or (at your option)\n"
+		"Foundation; either version 3 of the License, or (at your option)\n"
 		"any later version.\n"
 		"\n"
 		"gelide is distributed in the hope that it will be useful,\n"
